tests/unit/Handlers/PagesHandlerTest.php

```diff
@@ -79,6 +79,65 @@ public function testGetPageShouldReturnStatus404(): void
         $this->assertEquals(404, $response->getStatusCode());
     }
 
+    public function testFindPagesShouldReturnStatus200(): void
+    {
+        $request = Request::createFromEnvironment(
+            Environment::mock([
+                'REQUEST_METHOD' => 'POST',
+                'REQUEST_URI' => '/api/pages'
+            ])
+        )->withParsedBody([
+            'taxonomyFilter' => [
+                'taxonomyKey1' => ['taxonomyValue1'],
+                'taxonomyKey2' => ['taxonomyValue2', 'taxonomyValue3'],
+                'operation' => 'and'
```

Comment:
Shouldn't this be outside of the `taxonomyFilter`? An adjacent property?

Reply:
Fixed